spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,23 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
```

Comment:
@cynepco3hahue can you also add tests for unhappy path for authorization for this feature?